### PR TITLE
Update scripts.md

### PR DESCRIPTION
**Pull request by anthony-near**

### CODE REVIEW

File: scripts.md [hunk 1]
```diff
@@ -27,18 +32,23 @@
...
    ```sh
    near contract call-function as-transaction v2.nft.kagi.testnet ckt_approve_call json-args '{"token_id":"1","account_id":"canhazgas.testnet","msg":"{\"is_paymaster\":true}"}' prepaid-gas '100.0 Tgas' attached-deposit '1 yoctoNEAR' sign-as hatchet.testnet network-config testnet sign-with-legacy-keychain send
    ```
 
    **Mark key for use as paymaster**
+   - NOTE: call this once per `chain_id` you wish to add.
+   - ensure the `chain_id`, `token_id`, and `canhazgas.testnet` fields are updated to the appropriate values
 
    ```sh
    near contract call-function as-transaction canhazgas.testnet add_paymaster json-args '{"chain_id":"97","balance":"100000000000000000000","nonce":0,"token_id":"1"}' prepaid-gas '100.0 Tgas' attached-deposit '0 NEAR' sign-as canhazgas.testnet network-config testnet sign-with-legacy-keychain send
```
Comment (anthony-near) [inline scripts.md:55]: @encody does the `"balance":"100000000000000000000"` ever need to be changed or should it remain constant?

Comment (encody) [reply scripts.md:55]: It should be changed to the balance of the account on a chain at the time it is added as a paymaster for that chain. The balance must be specified in units of the smallest indivisible unit of gas token (i.e. wei on Ethereum mainnet).

Comment (encody) [reply scripts.md:55]: Also, `` `canhazgas.testnet` `` is probably a typo, as it is not the name of a field.

Comment (anthony-near) [reply scripts.md:55]: The `sign-as canhazgas.testnet` field or `as-transaction canhazgas.testnet add_paymaster` field?

File: scripts.md [hunk 2]
```diff
@@ -27,18 +32,23 @@
...
-
+   - ensure the `hatchet.testnet`, and `canhazgas.testnet` fields are updated to the appropriate values
+  
    ```sh
    near contract call-function as-transaction canhazgas.testnet add_administrator json-args '{"account_id":"hatchet.testnet"}' prepaid-gas '100.0 Tgas' attached-deposit '0 NEAR' sign-as canhazgas.testnet network-config testnet sign-with-legacy-keychain send
    ```
 
-   **Transfer chain key NFT**
+   **Transfer chain key NFT - if NFT already minted**
+     - NOTE: to mint NFT see https://github.com/near/multichain-gas-station-contract/tree/pyth-client/nft_key#creating-new-key-tokens
```
Comment (encody) [inline scripts.md]: This should probably be a relative link like this:

```md
[link](./nft_key#creating-new-key-tokens)
```

File: scripts.md (hunk 1 above)
Comment (encody) [reply scripts.md:55]: oh, I thought you were referring to the names of fields in the args JSON payload. My bad.

Comment (anthony-near) [reply scripts.md:55]: How do you know which foreign chain account is associated with NFT?

Comment (encody) [reply scripts.md:55]: Call [`nft_key.near->ckt_public_key_for(token_id)`](https://github.com/near/multichain-gas-station-contract/blob/0ad3dd68d1f53129b482eaae865bca1a2daedbb8/nft_key/src/lib.rs#L168) and then convert it into an address using something like [`ethers_core::utils::raw_public_key_to_address`](https://docs.rs/ethers-core/latest/ethers_core/utils/fn.raw_public_key_to_address.html) ([example](https://github.com/near/multichain-gas-station-contract/blob/0ad3dd68d1f53129b482eaae865bca1a2daedbb8/lib/src/foreign_address.rs#L20-L22)).

**Pull request merged**